annotation-definitions/src/main/java/eu/europeana/annotation/definitions/model/search/result/IdBean.java

```diff
@@ -3,21 +3,18 @@
 import java.util.Date;
 
 public interface IdBean {
-
-	/**
-	 * Retrieve the Europeana object unique Id
-	 * 
-	 * @return The Europeana object UniqueID
-	 */
-	String getIdAsString();
 	
 //	/**
 //	 * The date the record was created
 //	 * @return 
 //	 */
 //	Date getCreatedTimestamp();
 	
-	/**
+    String getIdentifierAsUriString();
```

Comment:
I think it is better to keep this name, and also the one below, the annotation id does not exist any more, only the identifier field. Also the getIdentifier() method (the one below you want to rename to) exists in the AbstractAnnotation which also implements the IdBean interface, which would cause some naming collisions.